src/hooks/useInactiveListener.ts

```diff
@@ -6,16 +6,28 @@ import { injected } from '../connectors'
  * Use for network and injected - logs user in
```

Comment:
we can delete this file right?

Reply:
yes we could actually. I'll do it in one of my next commits :)